src/include/duckdb/optimizer/join_order_optimizer.hpp

```diff
@@ -12,43 +12,35 @@
 #include "duckdb/common/unordered_set.hpp"
 #include "duckdb/optimizer/join_order/query_graph.hpp"
 #include "duckdb/optimizer/join_order/join_relation.hpp"
+#include "duckdb/optimizer/join_node.hpp"
 #include "duckdb/parser/expression_map.hpp"
 #include "duckdb/planner/logical_operator.hpp"
 #include "duckdb/planner/logical_operator_visitor.hpp"
+#include "duckdb/optimizer/cardinality_estimator.hpp"
 
 #include <functional>
 
+template <>
```

Comment:
was unsure where to put this template because I kept getting namespace errors. Feedback appreciated here!